app/assets/stylesheets/ama_layout/layout_components/banner.scss

```diff
@@ -0,0 +1,17 @@
+.banner{
+  position: relative; /* This is needed for the positioning of the close icon on the blue banners*/
+
+  &--blue{
+    background: $brand-blue-insurance;
+    padding: $base-padding;
+  }
+
+  &--red{
+    background: $brand-red;
+    padding: $base-padding;
+  }
+
+  &__text{
```

Comment:
Just to clarify - you guys did say its to be expected that some are &-- and some are &__ right?